examples/cerebras/chat.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\AI\Agent\Agent;
+use Symfony\AI\Platform\Bridge\Cerebras\ApiKey;
+use Symfony\AI\Platform\Bridge\Cerebras\Model;
+use Symfony\AI\Platform\Bridge\Cerebras\PlatformFactory;
+use Symfony\AI\Platform\Message\Message;
+use Symfony\AI\Platform\Message\MessageBag;
+
+require_once dirname(__DIR__).'/bootstrap.php';
+
```

Comment:
please check for the env var like in other examples, thanks

Reply:
yes, that's not needed anymore, but was moved into the `env(...)` function

Reply:
Right, sorry